### PR TITLE
Fix disappearing dudes in coop with respawn

### PR DESCRIPTION
**Pull request by CommonLoon102**
Related issue: https://github.com/nukeykt/NBlood/issues/196

Every change is just comment or magic number removal.
The fix is one line, negating the condition of an if. I marked it.

### CODE REVIEW

File: source/blood/src/actor.cpp [hunk 1]
```diff
@@ -6873,9 +6873,9 @@ bool actCheckRespawn(spritetype *pSprite)
             if (pXSprite->respawnPending == 1)
                 nRespawnTime = mulscale16(nRespawnTime, 0xa000);
             pSprite->owner = pSprite->statnum;
-            actPostSprite(pSprite->index, 8);
-            pSprite->flags |= 16;
-            if (pSprite->type >= kDudeBase && pSprite->type < kDudeMax)
+            actPostSprite(pSprite->index, kStatRespawn);
+            pSprite->flags |= kHitagRespawn;
+            if (!IsDudeSprite(pSprite))
```
Comment (CommonLoon102) [inline source/blood/src/actor.cpp]: This is the fix.

**Comment by nukeykt** [comment]
I see you referenced statnum names from **leaked** alpha code, so i can't accept your pull request. Just leave a fix

**Comment by CommonLoon102** [comment]
Actually, I've referenced them from BloodGDX.

**Comment by nukeykt** [comment]
BloodGDX is definitely based on alpha code. Also you can't use any code from GDX because it's licensed under GPL3+ whereas my code is GPL2+

**Comment by CommonLoon102** [comment]
Would it be OK if I change the macro names and comments?

**Comment by CommonLoon102** [comment]
Maybe the macro names can stay, as these were already in:
#define kStatRespawn 8
#define kStatMarker 10
#define kStatFree 1024

The leaked source has the same names.

**Comment by CommonLoon102** [comment]
Anyways, it doesn't even compile anymore :D
I'm closing this.

**Pull request closed without merging**